src/main/resources/freemarker/dataset-approved.html

```diff
@@ -22,7 +22,7 @@
         <tr>
             <td style="padding: 15px 15px 0px; font-family: 'Montserrat', sans-serif; font-size: 16px; color: #1F3B50; text-align: justify; line-height: 25px;">
                 <p id="content" style="margin: 0;">
-                    Thank you for using DUOS to register your dataset(s). We’re happy to inform you that ${dacName} has accepted your dataset. The ${datasetName} dataset should now be visible in the DUOS dataset catalog. Researchers will now be able to submit data access requests through DUOS for access to your dataset.
+                  Your dataset, ${datasetName}, submitted to the ${dacName} by ${dataSubmitterName} for management of future data access requests has been <b>accepted</b> and can be found in the DUOS Data Library with DUOS ID: ${datasetName}
```

Comment:
I think the phrase `submitted to the ${dacName} by ${dataSubmitterName} for management of future data access requests` is an appositive clause so it also should have commas around it, in addition to `datasetName` since it's also an appositive for `Your dataset`. Also maybe it should be split into two sentences with a period at the end.
```suggestion
                  Your dataset, ${datasetName}, submitted to the ${dacName} by ${dataSubmitterName} for management of future data access requests, has been <b>accepted</b>. It can be found in the DUOS Data Library with DUOS ID: ${datasetName}.
```

As I understand it, the rationale for separating the appositive clause like this is to make it easer for a reader to find the verb and direct object of a sentence.

Reply:
The language is specified by product - I'd prefer to leave this decision to @solideoglori for language choice in these cases.